src/libraries/System.Net.Http/src/System/Net/Http/Headers/HttpHeaders.cs

```diff
@@ -82,27 +82,26 @@ internal void Add(HeaderDescriptor descriptor, string value)
             }
         }
 
-        public void Add(string name, IEnumerable<string> values)
+        public void Add(string name, IEnumerable<string?> values)
         {
             Add(GetHeaderDescriptor(name), values);
         }
 
-        internal void Add(HeaderDescriptor descriptor, IEnumerable<string> values)
+        internal void Add(HeaderDescriptor descriptor, IEnumerable<string?> values)
         {
             if (values == null)
             {
                 throw new ArgumentNullException(nameof(values));
             }
 
-            HeaderStoreItemInfo info;
             bool addToStore;
```

Comment:
Can also move the `addToStore` declaration to the `out` parameter as well?

Reply:
sure